jamime/github-actions-public
# Update README.md

name: Bug Report
description: Something isn't working as expected?
labels:
  - Bug
  - triage
body:
  - type: markdown
    attributes:
      value: |
        ### ⚠️ This bug report is public
        Please create a reproducible example using this [CodeSandbox template](https://codesandbox.io/s/carbon-quickstart-xi5jc). 
        Please do not include any confidential or commercially sensitive information.

        ---
  - type: textarea
    id: current-behaviour
    attributes:
      label: Current behaviour
      description: |
        Please give a clear and concise description of the current behaviour.
        If applicable, add screenshots or screen recording of your CodeSandbox to help explain the problem. You can paste these directly into GitHub.
    validations:
      required: true
  - type: textarea
    id: expected-behaviour
    attributes:
      label: Expected behaviour
      description: |
        Please give a clear and concise description of the current behaviour.
    validations:
      required: true
  - type: input
    id: url
    attributes:
      label: CodeSandbox or Storybook URL
      description: >- 
        Please fork [this template](https://codesandbox.io/s/carbon-quickstart-xi5jc), and ensure you save your changes.
        Alternatively, if your issue relates to [Storybook](https://carbon.sage.com) please link to the story.
      placeholder: https://codesandbox.io/s/carbon-quickstart-xi5jc
    validations:
      required: true
  - type: textarea
    id: suggested-solution
    attributes:
      label: Suggested Solution
    validations:
      required: false
  - type: input
    id: carbon-version
    attributes:
      label: Carbon Version
      description: What version of carbon are you using?
      placeholder: 77.13.2
    validations:
      required: true
  - type: dropdown
    id: browsers
    attributes:
      label: What browsers are you seeing the problem on?
      multiple: true
      options:
        - Firefox
        - Chrome
        - Safari
        - Microsoft Edge
        - Other
    validations:
      required: true
  - type: dropdown
    id: os
    attributes:
      label: What Operating System are you seeing the problem on?
      multiple: true
      options:
        - MacOS
        - Windows
        - Linux
        - Android
        - iOS
        - Other
    validations:
      required: true
  - type: textarea
    id: additional-info
    attributes:
      label: Anything else we should know?
    validations:
      required: false
  - type: checkboxes
    id: terms
    attributes:
      label: Confidentiality
      description: >-
        By submitting this issue, you confirm that there is no confidential or commercially sensitive information included.
      options:
        - label: I confirm there is no confidential or commercially sensitive information included.
          required: true
